wtp/apiCaller.js

```diff
@@ -72,12 +69,12 @@ const runWtpTest = async (url, mobile, cb) => {
             width: config.get('wtp.viewportWidth'),
             height: config.get('wtp.viewportHeight'),
             custom: config.get('wtp.imageScript'),
-            location: 'Dulles:Chrome.Native', // Native means no speed shaping in browser, full speed ahead
-            mobile: (mobile) ? 1 : 0,
+            location: await locationSelector.getLocation() + ':Chrome.Native', // Native means no speed shaping in browser, full speed ahead
+      mobile: (mobile) ? 1 : 0,
             fvonly: 1, // first view only
             timeline: 1 // workaround for WPT sometimes hanging on getComputedStyle()
           },
-    headers: { 'User-Agent': 'WebSpeedTest', 'X-WPT-API-KEY': apiKey },
+    headers: { 'User-Agent': 'WebSpeedTest', 'X-WPT-API-KEY': apiKeys.get() },
```

Comment:
Maybe to keep the semantics of `key` we could rename `get` to `getRandom()` or `getKey()` if random is just an implementation detail..

Reply:
thanks, corrected as per suggestion